### ISSUE TITLE
findByClientID($id)

### ISSUE THREAD

**Issue by hjone72** [invoiceninja/sdk-php#12]
Hi,

Would it be possible to add a method that allows searching by ClientID? The below is working for my local install.

`Models/AbstractModel.php`
```
    public function findByClientID($id)
    {
        $url = sprintf('%s/?client_id=%s', static::getRoute(), $id);
        $data = static::sendRequest($url);

        return static::hydrate($data);
    }
```

Thanks,

**Comment by hillelcoren** [comment]
Sure, feel free to create a pull request

**Comment by hjone72** [comment]
This actually causes an issue if multiple results are returned.

For example;

`$credits = Credit::findByClientID(13); // Where clientID 13 has multiple credits an object like below would be returned.`

```
InvoiceNinja\Models\Credit Object
(
    [id] => 
    [0] => stdClass Object
        (
            [account_key] => 
            [is_owner] => 
            [id] => 23
            [amount] => 45
            [balance] => 45
            [updated_at] => 1496656509
            [archived_at] => 
            [is_deleted] => 
            [credit_date] => 2017-06-05
            [credit_number] => 
            [private_notes] =>
            [public_notes] => 
            [client_id] => 9
        )

    [1] => stdClass Object
        (
            [account_key] => 
            [is_owner] => 
            [id] => 17
            [amount] => 15
            [balance] => 0
            [updated_at] => 1496292488
            [archived_at] => 1496292488
            [is_deleted] => 
            [credit_date] => 2017-02-02
            [credit_number] => 
            [private_notes] =>
            [public_notes] => 
            [client_id] => 9
        )

)
```

This can be fixed by #17
